docker/base/kilda-base-lab-service/Dockerfile

```diff
@@ -37,19 +37,23 @@ RUN apt update \
         libtool-bin \
         gcc \
         build-essential \
-        automake \
-    && pip3 install setuptools==45.2.0 \
+        automake
+
+RUN pip3 install setuptools==45.2.0 \
```

Comment:
Here and in other places: pip can install multiple packages at once, just use `pip3 install package1==1.0.0 package2=0.1.0...`